tenant/storage_org.go

```diff
@@ -217,13 +218,13 @@ func (s *Store) UpdateOrg(ctx context.Context, tx kv.Tx, id platform.ID, upd inf
 		}
 
 		if err := idx.Delete(organizationIndexKey(u.Name)); err != nil {
-			return nil, ErrInternalServiceError(err)
+			return nil, errors.ErrInternalServiceError(err)
```

Comment:
In this file, you have some `ErrInternalServiceError` where you add the `errors.WithErrorOp` and some that you don't. Should they be consistent?

Reply:
Apparently not, judging by the existing code.  Some errors in, for example, `GetOrgByName`, return [op codes](https://github.com/influxdata/influxdb/blob/8e8700f14f8b45601f59cb20908ee5baa8667328/tenant/error_org.go#L30) for that [call](https://github.com/influxdata/influxdb/blob/8e8700f14f8b45601f59cb20908ee5baa8667328/tenant/storage_org.go#L98C21-L98C21) but some of the actions within `CreateOrg` have their own [private op codes](https://github.com/influxdata/influxdb/blob/8e8700f14f8b45601f59cb20908ee5baa8667328/tenant/error_org.go#L42), like `kv/MarshalOrg`.  And some OS errors bubble up without op codes.

I need to consider how much time we want to spend on rationalizing this...